tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/ExtensionMethodsSuite.scala

```diff
@@ -433,7 +435,40 @@ class ExtensionMethodsSuite extends BaseDottySuite {
     )
   }
 
-  final val defcrc = Defn.Def(Nil, tname("crc"), Nil, Nil, Some(pname("Int")), int(2))
+  test("extension quasiquotes 1") {
+    import dialects.Scala3
+
+    val q"extension [..$tparams](...$params) { ..$stats }" =
```

Comment:
Could we add a tests with `tparams`?

Reply:
added. the code parsing tparams is the same as elsewhere, unlike the paramss parsing, but you are right, tests don't need to know that.

Reply:
Thanks!